bin/blocks-dump

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser
+
+from blocks.scripts import dump
+
+if __name__ == "__main__":
+    parser = ArgumentParser("Dumps your pickled main loop")
```

Comment:
Could eventually do with a bit more documentation e.g. telling the user that this extracts the parameters in NumPy files and allows the model to be shared with people on other platforms.